test/cli/install/registry/bun-install-registry.test.ts

```diff
@@ -14,25 +14,25 @@ var testCounter: number = 0;
 var port: number = 4873;
 var packageDir: string;
 
-beforeAll(async () => {
-  verdaccioServer = fork(
-    await import.meta.resolve("verdaccio/bin/verdaccio"),
-    ["-c", join(import.meta.dir, "verdaccio.yaml"), "-l", `${port}`],
-    { silent: true, execPath: "bun" },
-  );
-
-  await new Promise<void>(done => {
-    verdaccioServer.on("message", (msg: { verdaccio_started: boolean }) => {
-      if (msg.verdaccio_started) {
-        done();
-      }
-    });
-  });
-});
-
-afterAll(() => {
-  verdaccioServer.kill();
-});
+// beforeAll(async () => {
```

Comment:
was disabling verdaccio (and subsequently, these tests) intentional?

Reply:
Not intentional, for testing locally